docs/spec_draft.md

```diff
@@ -941,6 +942,63 @@ produces an implementation-defined value.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.dynamic_slice
+
+### Semantics
+
+Extracts a subtensor of the shape in `slice_sizes` from the `operand` tensor
+starting from `start_indices` and produces a `result` tensor. More formally,
+`result[i0, ..., iR-1] = operand[j0, ..., jR-1]` where
+`jd = start_indices[d] + id`.
+
+### Inputs
+
+| Name            | Type                                                     |
+|-----------------|----------------------------------------------------------|
+| `operand`       | tensor of any supported type                             |
+| `start_indices` | variadic number of 0-dimensional tensors of integer type |
```

Comment:
of type `si64` and keep the verification revisit with the open ticket https://github.com/openxla/stablehlo/issues/557

Reply:
Both HLO and MHLO agree on indices being tensors of integer type. Any particular reason to restrict that to `si64`?

Reply:
I agree with your point. That was an oversight.